data/quickstarts/gpu-quickstart.yaml

```diff
@@ -0,0 +1,45 @@
+kind: ConsoleQuickStart
+metadata:
+  name: gpu-quickstart
+  annotations:
+    opendatahub.io/categories: 'Getting started,GPUs,Python'
+spec:
+  displayName: Installing and verifying NVIDIA GPU add-on
+  appName: nvidia
+  durationMinutes: 5
+  icon: 'images/nvidia.svg'
+  description: Install GPU add-on
+  introduction: |-
+    ### This quick start shows you how to install the GPU add-on and verify that the JupyterHub spawner detects the available GPUs.
+  tasks:
+    - title: Launch Red Hat Hybrid Cloud Console
+      description: |-
+        1. Navigate to the Red Hat Hybrid Cloud Console in a new tab on your web browser.
+        2. On the left navigation menu select **Clusters** and select the cluster you are working on.
+        3. From the Cluster Overview page, navigate to the **Add-ons** menu.
+        3. Install the **NVIDIA GPU Add-on**.
+
+      review:
+        instructions: |-
+          #### To verify you have installed the GPU add-on:
+          Can you see the add-on with the label **Installed** in the add-ons menu?
+        failedTaskHelp: This task is not verified yet. Try the task again.
+      summary:
+        success: You have installed the NVIDIA GPU Add-on.
+        failed: Ensure you have met the prerequisites for the GPU add on and try the steps again.
+    - title: Verify that the JupyterHub spawner sees the available GPUs
+      description: |-
+        1. Open the JupyterHub spawner from the Red Hat OpenShift Data Science dashboard.
+        2. For the Notebook image, the options **CUDA**, **Pytorch**, **Tensorflow** should be available to launch.
+        3. Select a deployment size based on your needs. If you have more than 1 GPU, select the number of GPUs that JupyterHub should use.
```

Comment:
Even if you have only 1, you still have the choice between 0 and 1

Reply:
ack, changed it accordingly

Reply:
I don't see this indication anymore? I think it should still be there, but it should not say `If you have more than 1[...]`